lib/project_types/script/layers/application/build_script.rb

```diff
@@ -9,11 +9,14 @@ def call(ctx:, task_runner:, script_project:, library:)
             CLI::UI::Frame.open(ctx.message("script.application.building")) do
               begin
                 UI::StrictSpinner.spin(ctx.message("script.application.building_script")) do |spinner|
+                  metadata_file_location = task_runner.metadata_file_location
+                  metadata = Infrastructure::MetadataRepository.new(ctx: ctx).get_metadata(metadata_file_location)
```

Comment:
I'm just curious: what is the advantage of creating a `MetadataRepository` versus using the `TaskRunner`?

Reply:
This avoids the duplication between the different language task_runners and their tests. We already had some with just the AssemblyScript and TypeScript runners, and with the creation of the WasmTaskRunner I figured I could clean this up while I was here.

This actually came up during our pairing session @ErinRenNumber1 so I would love to get your 👀 ✅  here as well.